eZ/Publish/API/Repository/UserService.php

```diff
@@ -376,4 +377,8 @@ public function newUserGroupUpdateStruct();
      * @return \eZ\Publish\SPI\FieldType\ValidationError[]
      */
     public function validatePassword(string $password, PasswordValidationContext $context = null): array;
+
+    public function createPasswordHash(string $password, int $hashType = APIUser::DEFAULT_PASSWORD_HASH): string;
+
+    public function getServiceSettings(): array;
```

Comment:
```suggestion
    public function getPasswordHashType(): int;
```
Other settings than `hashType` are not used outside of UserService, so there is no need to expose it.